src/dry-command-interceptor.ts

```diff
@@ -0,0 +1,51 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import Process = NodeJS.Process;
+import { Cli } from './cli';
+import { NpmCommandProxy } from './npm-command-proxy';
+
+/**
+ * Responsible for the requested dry specific command interception
+ */
+export class DryCommandInterceptor {
+    private readonly rawArgs: string[];
+    private npmCommandProxy: NpmCommandProxy;
+    private drySpecificArgs: string[] = ['saveTo'];
+    /**
+     * @param {Cli} cli The CLI to use
+     * @param {NodeJS.Process} process The main process
+     */
+    constructor(cli: Cli, process: Process) {
+        this.rawArgs = process.argv.slice(2);
+        this.npmCommandProxy = new NpmCommandProxy(cli, this.rawArgs);
+    }
+
+    /**
+     * Propagate the command received by dry to npm
+     * @return {Promise<void>} Resolved promise on success, rejected promise on failure.
+     */
+    public proxy(): Promise<void> {
+
+        const intercept: boolean = this.rawArgs.length > 0 && this.drySpecificArgs.indexOf(this.rawArgs[0]) > -1;
```

Comment:
Now that we use an option prefixed dry instead of a command:
- the option could be any argument
- the npm proxy will be always called

Reply:
Ok, done. The packager proxy is called only if it exists arguments not already handled by the dry command or shared